website/docs/r/codepipeline.markdown

```diff
@@ -71,13 +71,20 @@ resource "aws_iam_role_policy" "codepipeline_policy" {
 EOF
 }
 
+data "aws_kms_alias" "s3kmskey" {
+  name = "alias/myKmsKey"
+}
+
 resource "aws_codepipeline" "foo" {
   name     = "tf-test-pipeline"
   role_arn = "${aws_iam_role.foo.arn}"
 
   artifact_store {
     location = "${aws_s3_bucket.foo.bucket}"
     type     = "S3"
+    encryption_key {
+      id   = "${data.aws_kms_alias.s3kmskey.arn}"
+      type = "KMS"
   }
```

Comment:
I think there's one bracket `}` missing here 😉

Reply:
you are correct! fixed.